src/error.h

```diff
@@ -187,6 +189,17 @@ extern Obj RACErrorHelper(int          mayReturn,
     RequireArgumentCondition(funcname, op, argname, IS_INTOBJ(op), \
         "must be a small integer")
 
+/****************************************************************************
+**
+*F  GetSmallInt
+*/
+static inline Int GetSmallInt(const char * funcname, Obj op, const char * argname)
+{
+    RequireArgumentCondition(funcname, op, argname, IS_INTOBJ(op),
+                             "must be a small integer");
```

Comment:
Any reason not to use `RequireSmallInt` here?

Reply:
Nope, changed